README.md

```diff
@@ -1,28 +1,92 @@
-# Jina Documentations
+# Jina AI Documentation
```

Comment:
is not Jina AI the company name and not the product?

Reply:
Either works tho?

Reply:
We agreed, that the product is "Jina" and the company is "Jina AI". Since this is the documentation for the product, I'd also remove the `AI` again.

Reply:
Ok, will do!